handbook/marketing/index.md

```diff
@@ -24,3 +24,16 @@ See job descriptions and responsibilities of roles on the Marketing team:
 - [Developer Advocate](roles.md#developer-advocate)
 - [VP Marketing](roles.md#vp-marketing)
 - [Online Marketer](roles.md#online-marketer)
+
+## Definitions
+
+### MQL (marketing-qualified lead)
+
+A marketing qualified lead is any of:
+
+- a submission of the request a demo, contact us, automation, or hackathon forms
```

Comment:
+ A person who fills out a demo request form
+ A person who requests a free trial
+ A person who fills out and form requesting to speak with a Sourcegraph representative